src/controllers/timeEntryController.js

```diff
@@ -140,7 +140,7 @@ const timeEntrycontroller = function (TimeEntry) {
           await initialTask.save();
         }
 
-        if (req.body.isTangible === true) {
+        if (req.body.isTangible === "true") {
```

Comment:
Main request body property "isTangible" should be a type of boolean, and not a string.

It will be nice to update the request handlers and implement the request from the frontend to pass the "isTangible" property as a boolean that will have either `true || false` value.

Reply:
Yes, ideally it would make more sense to be a boolean, but that is outside the scope of this PR. If you want, you can raise a bug in the bugs doc for fixing the data type in the API call that creates the time entries.